languages.json

```
@@ -1,5 +1,13 @@
 {
   "languages": {
+    "_8th": {
```

I think it would be better to rename this, and move it into the E section
```suggestion
    "Eighth": {
```

---

Good idea!